src/IO.ts

```diff
@@ -1,6 +1,74 @@
 /**
- * @file `IO<A>` represents a synchronous computation that yields a value of type `A` and **never fails**.
- * If you want to represent a synchronous computation that may fail, please see `IOEither`.
+ * @file `IO<A>` represents a synchronous computation that yields a value of type `A` and **never fails**.  If you want
+ * to represent a synchronous computation that may fail, please see `IOEither`.  IO actions are terminated by calling
+ * their `run()` method that executes the computation and returns the result.
+ *
+ * ```ts
+ * import { IO, io } from 'fp-ts/lib/IO'
+ *
+ * const constant: IO<number> = io.of(123);
+ * constant.run()  // returns 123
+ *
+ * const random: IO<number> = new IO(() => Math.random())
```

Comment:
Similar to @giogonzo's comment on `Console.ts`, you could mention [Random](https://github.com/gcanti/fp-ts/blob/master/src/Random.ts). Maybe the mention of these two would make sense after the examples or individually just before you introduce them.